.github/workflows/trigger-workflow.yaml

```diff
@@ -0,0 +1,31 @@
+name: Trigger Workflow
```

Comment:
Comments in this workflows whose the sole purpose is to test the actions would be helpful,

Reply:
Done, thanks.